modules/juce_core/system/juce_PlatformDefs.h

```diff
@@ -57,13 +57,16 @@ namespace juce
 #endif
 
 //==============================================================================
-#if JUCE_IOS || JUCE_LINUX
+#if JUCE_IOS
   /** This will try to break into the debugger if the app is currently being debugged.
       If called by an app that's not being debugged, the behaviour isn't defined - it may
       crash or not, depending on the platform.
       @see jassert()
   */
   #define JUCE_BREAK_IN_DEBUGGER        { ::kill (0, SIGTRAP); }
+#elif JUCE_LINUX
+  #include <cstdlib>
+  #define JUCE_BREAK_IN_DEBUGGER        { std::abort(); }

```

Comment:
As noted in the pull request description, `std::abort()` will result in `raise(SIGABRT)`, which will force the program to end. It would be better to use `raise(SIGTRAP)` here.

Reply:
It makes sense.

Reply:
Changed and tested on Arch Linux latest version